<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportListLeavedClient</class>
 <widget class="QDialog" name="ReportListLeavedClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="13" column="0" colspan="4">
    <widget class="QGroupBox" name="grpFilterColumns">
     <property name="title">
      <string>Отчет строится по столбцам</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0">
       <widget class="QCheckBox" name="chkOrderEvent">
        <property name="text">
         <string>Порядок оказания помощи</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="chkOrgReceived">
        <property name="text">
         <string>Кем направлен</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkAddress">
        <property name="text">
         <string>Адрес</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkFIO">
        <property name="text">
         <string>ФИО</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkExternalId">
        <property name="text">
         <string>№ Истории болезни</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="chkDiagnosis">
        <property name="text">
         <string>Диагноз</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="chkSetDate">
        <property name="text">
         <string>Дата поступления</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="chkMovingDate">
        <property name="text">
         <string>Дата поступления в отделение</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkLeavedDate">
        <property name="text">
         <string>Дата выписки</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="chkProfile">
        <property name="text">
         <string>Профиль</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="chkOperation">
        <property name="text">
         <string>Код операции</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="chkDays">
        <property name="text">
         <string>Проведено койко-дней</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="chkResult">
        <property name="text">
         <string>Исход</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="chkOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Тип госпитализации</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Категория пациентов</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkEventType">
     <property name="text">
      <string>Тип обращения</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="CRBListBox" name="lstEventType"/>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="CRBListBox" name="lstOrgStructure"/>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QComboBox" name="cmbHospitalization">
     <item>
      <property name="text">
       <string>не задано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Круглосуточная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Дневная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Отказ</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QComboBox" name="cmbTypeRegistry">
     <item>
      <property name="text">
       <string>Все</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Иностранцы</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Иностранцы с регистрацией</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Жители Санкт-Петербурга</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Жители Лен. области</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Иногородние</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes</header>
  </customwidget>
  <customwidget>
   <class>CRBListBox</class>
   <extends>QListView</extends>
   <header>library/RBListBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportListLeavedClient</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportListLeavedClient</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
